assets/scss/base.scss

```diff
@@ -361,73 +299,12 @@ a.v-btn {
 }
 // Common button end
 
-.h-divider {
-  margin: 0;
-  border-color: #e2e8ee;
+.theme--light.v-divider.h-divider {
+  border-width: 0 0 3px 0;
```

Comment:
We only want a bottom border. (Could have been top, too, but whatever.)

The previous CSS, `border-width: 3px`, created a 3px border on all 4 sides. Not what we wanted.